qa/tasks/cephfs/test_misc.py

```diff
@@ -232,35 +233,63 @@ def test_fs_lsflags(self):
         self.assertEqual(lsflags["allow_multimds_snaps"], True)
         self.assertEqual(lsflags["allow_standby_replay"], True)
 
-    def test_filesystem_sync_stuck_for_around_5s(self):
-        """
-        To check whether will the fsync will be be stuck to wait for the mdlog to be
-        flushed for at most 5 seconds.
-        """
+    def _test_sync_stuck_for_around_5s(self, dir_path, file_sync=False):
 
-        dir_path = "fsync_do_not_wait_mdlog_testdir"
         self.mount_a.run_shell(["mkdir", dir_path])
 
+        pyscript = dedent("""
```

Comment:
Could we rename `pyscript` to indicate sync directory, something like `sync_dir_pyscript` ?